app/src/androidTest/java/com/github/h3lp3rs/h3lp/signin/SignedInUserTest.kt

```diff
@@ -49,7 +47,10 @@ class SignedInUserTest : H3lpAppTest() {
         userSignIn.setString(globalContext.getString(R.string.KEY_USER_UID), "")
```

Comment:
Btw, I know this is not your code but this line is DANGER!
Indeed, if we return the default, the application will crash with `Could not parse HashMap to string`